### DISCUSSION TITLE
sahil2801/replit-code-instruct-glaive result

### DISCUSSION THREAD

**Discussion by regularfry** [abacaj/code-eval#4] [General]
Am I reading this right?  Right now, https://huggingface.co/sahil2801/replit-code-instruct-glaive, a 3B model, is claiming a pass@1 score of 63.5%, *significantly* better than the next best model, which is a 15B model.

Is that... real?  Or are we seeing an artefact of how the pass@1 test works?

Apologies if this is a stupid question, but that result just looks too good to be true.

**Comment by matthiasgeihs** [comment]
Have asked myself the same question and ran some validation experiments.
Seems like test data might have leaked into training data.
See [hf discussion](https://huggingface.co/sahil2801/replit-code-instruct-glaive/discussions/3#64a6dae2b869cc3dd67017e7) for more details.

**Comment by regularfry** [comment]
Hm. That's a pain.  These things happen.
